section/Init.cpp

```diff
@@ -0,0 +1,10 @@
+#include "moho.h"
+
+extern const uintptr_t FIRST_CTOR;
```

Comment:
Is the behavior of this documented? As in, what does this new file do?

Reply:
This file calls global variables constructors.